modules/graph_ql/main.tf

```diff
@@ -68,4 +77,17 @@ resource "aws_ecs_task_definition" "graphql_task" {
   }
 ]
 DEFINITION
+}
+
+resource "aws_ecs_service" "graphql_service" {
+  name            = "${var.prefix}_graphql_service"
+  cluster         = var.ecs_cluster_id
+  task_definition = aws_ecs_task_definition.graphql_task.arn
+  desired_count   = 3
```

Comment:
this means 3 instances will keep running all time. Should we lower this to 2 to save cost?

Reply:
3 is good, you always want an odd number to maintain availability.